### DISCUSSION TITLE
TypeError: UserList.__new__() got an unexpected keyword argument 'type'

### DISCUSSION THREAD

**Discussion by bulletproof2k** [Taxel/PlexTraktSync#1098] [Support] [answered]
When running the script, this error occurs. I use a shortcut to launch it. Perhaps I did something wrong during the installation process.
![Без имени1](https://user-images.githubusercontent.com/1818579/194902916-4eb8acff-ffda-45c7-959e-e6175187c4d1.jpg)

**Comment by simonc56** [comment, marked as answer]
#941
